feature/quicksettings/src/main/java/com/google/jetpackcamera/feature/quicksettings/QuickSettingsScreen.kt

```diff
@@ -81,6 +82,13 @@ fun QuickSettingsScreenOverlay(
         )
 
     if (isOpen) {
+        val onBack = {
+            when (shouldShowQuickSetting) {
+                IsExpandedQuickSetting.NONE -> toggleIsOpen()
+                else -> shouldShowQuickSetting = IsExpandedQuickSetting.NONE
+            }
```

Comment:
nit: Can you just put this inline in `BackHandler`?

```
BackHandler {
    when (shouldShowQuickSetting) {
        IsExpandedQuickSetting.NONE -> toggleIsOpen()
        else -> shouldShowQuickSetting = IsExpandedQuickSetting.NONE
    }
}
```

Reply:
onBack is used again below so I made it into a val

Reply:
I only see it being used in the `BackHandler` below. Where else is it being used?